modules/node-certificates.adoc

```diff
@@ -0,0 +1,29 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
+
+== Purpose
+
+The Kubelet, in 4.x, uses the bootstrap certificate located in
```

Comment:
Consider moving this to its own section on bootstraping?
In short, the Node certificates are signed by the cluster; but they come from a CA' that is generated by the bootstrap process. Having what you have here is good, but it doesn't really talk about Node certificates; it focuses on the bootstrap certificates.